app/src/lib/components/Experience/History/Box.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  startDate?: string;
+  endDate?: string;
+  title?: string;
+  des?: string;
+}
+
+const Box = (props: Props) => {
+  const { startDate, endDate, title, des } = props;
+
+  return (
+    <div>
+      <Wrap>
+        <div className="intro">
+          <div className="date">
+            <div className="start-date">{startDate}</div>
+            <div className="end-date">{endDate}</div>
```

Comment:
```suggestion
            <span className="start-date">{startDate}</span>
            <span className="end-date">{endDate}</span>
```
이 부분만 얘기하는건 아닌데 우리 조금더 시맨틱한 태그를 써봅시다

Reply:
화긴 후 시맨틱 하게 수정했습니다이